src/util/storage.native.ts

```diff
@@ -0,0 +1,21 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+const storage = {
```

Comment:
Can you use `export const storage` instead of exporting `{ storage }`? That would be a bit more standard.

Reply:
Fixed